src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -41,11 +42,17 @@ public Lobby(Long id, String name, Visibility visibility) {
     }
 
     @Override
-    public byte[] generateQrCode(String code) {
-        String data = URLEncoder.encode(HANNIBAL_URL + code, StandardCharsets.UTF_8);
+    public void generateQrCode() throws RestClientException {
+        String data = URLEncoder.encode(HANNIBAL_URL+invitationCode, StandardCharsets.UTF_8);
         RestTemplate restTemplate = new RestTemplate();
+        String QR_API_URL = "https://api.qrserver.com/v1/create-qr-code";
```

Comment:
why not private final static (see above)?
This URL won't change.

Reply:
It is in the scope of the method, so it can't be accessed anyway. But I can add it